README.md

```diff
@@ -1,105 +1,103 @@
-    - [Prerequisites](#prerequisites)
-    - [Setup](#setup)
-    - [Install](#install)
-    - [Usage](#usage)
-  - [🔭 Future Features ](#future-features)
-  - [👥 Authors ](#author)
-  - [Contributions](#contributions)
-  - [⭐️ Show your support ](#️support)
-  - [🙏 Acknowledgments ](#acknowledgments)
-  - [📝 License ](#license)
-
-
-# 📖 Math Magicians  <a name="React"></a>
-**To Do List** is a project under microverse Module 2 block 2 aimed at testing our knowledge about the concepts learnt and to help us build an application to take in and display tasks.
-## 🛠 Built With <a name="build-with"></a>
-
-**React, Redux**
-
-### Tech Stack <a name="tech-stack"></a>
-
-**React, Redux**
-
-### Key Features <a name="key-features"></a>
-- **Base configuration for math SPA**
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
- 
-
-## 💻 Getting Started <a name="getting-started"></a>
-
-To get a local copy up and running, follow these steps.
-
-### Prerequisites <a name="prerequisites"></a>
-
-In order to run this project you need: React, Redux, npm, node.js
-### Setup <a name="setup"></a>
-
-Clone this repository to your desired folder: 
--git clone https://github.com/Rebelzob/Math-Magicians.git
-
-### Install <a name="install"></a>
-
-Install the required dependencies by running:
-npm i -y
-
-### Usage <a name="usage"></a>
-
-To run the project, execute the following command:
--npm start
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-
-## 🔭 Future Features <a name="future-features"></a>
-
--  **Add more functionality to the calculator**
-
-## 👥 Author <a name="authors"></a>
-
-👤 **Karim Barragan**
-
-- GitHub: [@Rebelzob](https://github.com/Rebelzob)
-- LinkedIn: [Karim-Barragan](https://www.linkedin.com/in/karim-barragan/)
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-## Contributions
-
-Contributions, issues, and feature requests are welcome!
-
-
-## ⭐️ Show your support <a name="support"></a>
- 
-If you like this project like it and give me a follow.
-
-
-## 🙏 Acknowledgments <a name="acknowledgements"></a>
-
-I would like to thank Microverse for the oportunity given to me to work in this project
-
-
-## 📝 License <a name="license"></a>
-
-This project is [MIT](./LICENSE) licensed.
-
-
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <h1><b>Math Magicians 🧮</b></h1>
+  <br/>
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 [Math Magicians] ](#React)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#getting-started)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [🔭 Future Features ](#future-features)
+  - [👥 Authors ](#author)
+  - [Contributions](#contributions)
+  - [⭐️ Show your support ](#️support)
+  - [🙏 Acknowledgments ](#acknowledgments)
+  - [📝 License ](#license)

```

Comment:
- [ ] Please ensure that clicking on the following links in the `table of contents` takes the user to the respective parts of the readme file for a more professional readme file. Currently, these links don't work making them broken.

![image](https://github.com/Rebelzob/Math-Magicians/assets/81932478/8dcb0447-93be-4dc7-bf91-a8834be82923)
![image](https://github.com/Rebelzob/Math-Magicians/assets/81932478/c7a64913-f9d4-407d-8b61-18f565d9e852)
![image](https://github.com/Rebelzob/Math-Magicians/assets/81932478/78d8f458-8b9f-4092-9533-6c67de882a48)
![image](https://github.com/Rebelzob/Math-Magicians/assets/81932478/d85ed425-2461-412b-b976-4ce802cfa11c)